### PR TITLE
[SELS_TASK] [BE] SIGN UP - Create endpoint for register or adding the user

### PR DESCRIPTION
**Pull request by presyos**
[Asana Link]

https://app.asana.com/0/1203098252776160/1203104625958489/f

[Commands]

Backend

Setup the virtual environment
```
cd backend
.\venv\Scripts\Activate
```

Run the application
`python manage.py runserver`

### CODE REVIEW

File: backend/elearning/settings.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import environ
-
 from pathlib import Path
 
+import environ
+
 env = environ.Env()
```
Comment (Alvilb) [inline backend/elearning/settings.py:5]: be careful when rebasing next time, since we may get some unnecessary changes like this

File: backend/elearning/settings.py [hunk 2]
```diff
@@ -25,57 +25,61 @@
...
+    "django.contrib.staticfiles",
     # third party packages
-    
+    "corsheaders",
+    "rest_framework",
+    "rest_framework.authtoken"
     # local apps
-    'learnings'
+    "learnings",
 ]
```
Comment (Alvilb) [inline backend/elearning/settings.py:40]: fix spacing
```
[
# django apps
django_apps

# third party packages
third_party packages

# local apps
local_apps

File: backend/learnings/views.py [hunk 3]
```diff
@@ -1,3 +1,29 @@
-from django.shortcuts import render
+from django.contrib.auth.models import User
+from rest_framework import generics
```
Comment (Alvilb) [inline backend/learnings/views.py:2]: add space here

Comment (Alvilb) [reply backend/learnings/views.py:2]: for now lets proceed with this, later or tomorrow ill create a config file for isort to fix this :bow:

File: backend/elearning/settings.py (hunk 1 above)
Comment (presyos) [reply backend/elearning/settings.py:5]: moved by linter/isort

File: backend/elearning/settings.py (hunk 2 above)
Comment (presyos) [reply backend/elearning/settings.py]: moved by linter/isort
